packages/twig/src/patterns/components/button/button.wingsuit.yml

```diff
@@ -71,8 +71,8 @@ button:
       description: The disabled state of the button.
       required: false
       options:
-        true: "true"
-        false: false
+        "true": "true"
```

Comment:
The true value in this case is a string rather than a boolean. I think all these should be booleans right?

Reply:
There are several more instances of this. and a minor thing which is that true / false is sometimes capitalized and other times now 😆

Reply:
yes, it's all a bit messy, I'll fix this by making them coherent, I'll make the labels as strings, with capital letters, so like this:

```
        "true": True
```